fallout/items/misc/table_decorations.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/table_flag
+	name = "table flag"
+	desc = "Want to be patriotic, but don't have the space for a full flag? Boy, do we have the solution for you!"
+	icon = 'fallout/icons/objects/decorative.dmi'
+
+/obj/item/table_flag/pole
+	name = "table flagpole"
+	desc = "An empty flagpole. You feel less patriotic just looking at it."
+	icon_state = "flagpole"
+
```

Comment:
Why not make the blank pole the base type?
Also, stick f13 somewhere in the pathing, please.